keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -603,6 +586,8 @@ Files:
 - test/integration/servicecidr/allocator_test.go
 - test/integration/servicecidr/migration_test.go
 - test/integration/servicecidr/servicecidr_test.go
+- test/integration/servicecidr/feature_enable_disable_test.go
+- test/integration/servicecidr/perf_test.go
```

Comment:
Can we get links to https://storage.googleapis.com/k8s-triage/index.html for all the integration tests?

Reply:
yep, added a new link with a regex to cover all the linked tests